docs/set_up_tails.rst

```diff
@@ -103,6 +85,11 @@ Some other things to keep in mind:
    passphrases. We recommend using `spaced-repetition
    <https://en.wikipedia.org/wiki/Spaced_repetition>`__  to memorize
    Diceware passphrases.
+   
+.. note:: Tails doesn't always completely shut down and reboot
+	  properly when you click "restart", so if you notice a
+	  significant delay, you may have to manually power off and
+	  restart your computer for it to work properly.
```

Comment:
This note dates back to 2015 (a Trevor Timm/James Dolan collaboration no less - c311d272ff24f20588695982d87bbcbec4919ee7), I'm curious if this is actually still an intermittent issue that deserves calling out. But that's beyond the scope of this PR.